docs/Getting-Started-GPU.md

```diff
@@ -27,18 +26,28 @@ pip install --upgrade "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-re
 ### Install Levanter
 
 ```bash
+git clone https://github.com/stanford-crfm/levanter.git
 cd levanter
 pip install -e .
 ```
 
+### wandb login
+
+To use wandb to track your learning curves and other experiment metrics, you con log in to your wandb account on the command line as follows:
+```bash
+wandb login ${YOUR TOKEN HERE}
+```
+You can find more information on getting setup with weights and biases here: https://wandb.ai/site
+
+
 ## Running a job on Slurm
 
 ### Single Node
 
-Simple example, customize to your needs:
+A simple srun example within your levanter conda environment and cloned levanter directory. Customize to your needs:
 
 ```bash
-srun --cpus-per-task=128 --gres=gpu:8 --job-name=levanter-multi-1 --mem=1000G --nodelist=sphinx7 --open-mode=append --partition=sphinx --time=14-0 ~/src/levanter/scripts/run-slurm.sh python src/levanter/main/train_lm.py --config_path config/gpt2_small.yaml
```

Comment:
This command was also giving me some issues and I thought an example from the examples dir might be fitting here. Especially now that we'll have a whole doc on customizing this example to the user's needs.

Reply:
Okay, I'll switch it back